infra/base-images/base-runner/Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY --from=base-clang /usr/local/bin/llvm-cov /usr/local/bin/
 COPY --from=base-clang /usr/local/bin/llvm-profdata /usr/local/bin/
 COPY --from=base-clang /usr/local/bin/llvm-symbolizer /usr/local/bin/
 
+RUN apt-get update
```

Comment:
`base-image` does this command already. Are you sure we need it here as well?

Reply:
I do not think so, this must have been a debug leftover

Reply:
thanks for verifying that it's needed!